lisp/ess-custom.el

```diff
@@ -1501,6 +1501,9 @@ line starts with the roxy prefix.")
 
 ;; SJE -- this should not be defcustom - user does not set it.
 (defvaralias 'ess-current-process-name 'ess-local-process-name)
+
+;; should this have been make-buffer-local-variable?
+;; we always setq-local `ess-local-process-name' per buffer
```

Comment:
Since you are not a core developer, can you ask questions in an issue or github comment rather than in the source code?